app/views/hyrax/homepage/_home_content.html.erb

```diff
@@ -1,6 +1,12 @@
 <div class="col-md-6 column-extra-padding">
 
-  <p class="lead text-center">This is the prototype for the Duke University Libraries Research Data Repository.</p>
+  <p class="lead text-center">This is the prototype for the Duke University Libraries Research Data Repository What.</p>
```

Comment:
Why "What" at the end of this line?

Reply:
Doh! Thanks for catching that. I must have typed in the wrong window and forgot to fix it.